man/systemd.special.xml

```diff
@@ -949,20 +948,29 @@
         <varlistentry>
           <term><filename>network.target</filename></term>
           <listitem>
-            <para>This unit is supposed to indicate when network
-            functionality is available, but it is only very weakly
-            defined what that is supposed to mean, with one exception:
-            at shutdown, a unit that is ordered after
-            <filename>network.target</filename> will be stopped before
-            the network — to whatever level it might be set up then —
-            is shut down. It is hence useful when writing service files
-            that require network access on shutdown, which should order
-            themselves after this target, but not pull it in. Also see
-            <ulink url="https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget">Running
-            Services After the Network is up</ulink> for more
-            information. Also see
-            <filename>network-online.target</filename> described
-            above.</para>
+            <para>This unit is supposed to indicate when network functionality is available, but it is only
+            very weakly defined what that is supposed to mean. However, the following should apply at
+            minimum:</para>
+
+            <itemizedlist>
+              <listitem><para>At start-up, any configured synthetic network devices (i.e. not physical ones
+              that require hardware to show up and be probed, but virtual ones like bridge devices and
+              similar which are created programmatically) that do not depend on any underlying hardware
```

Comment:
Hmm, `networkd` start to create virtual interfaces but does not guarantee that to be created.

Reply:
I think it's actually enough that the netlink packets to create the virtual interfaces are enqueued, since the kernel will process netlink requests in order (at least i think). i.e. if some other program queries netlink to list interfaces afterwards, these requests be processed after the netlink packets that created the virtual interfaces, hence we should be good.

(That said, I think it would be nicer if we'd indeed wait for the netlink replies for the create requests)